src/backend/interfaces/NATS/py.client/caliopen.yaml.template

```diff
@@ -0,0 +1,32 @@
+delivery_agent:
```

Comment:
this file should not be duplicated for docker build purpose.

As we do a complete get of repository in the container image, the /src/backend/configs/caliopen.yaml.template exist and a copy of it in the correct directory should do the job instead of such duplication